### PR TITLE
Update carbon intensity data

### PR DESCRIPTION
**Pull request by benoit-cty**
Update carbon intensity data for cloud providers.
Remove AWS and Azure as they do not provide data.

### CODE REVIEW

File: codecarbon/core/emissions.py [hunk 1]
```diff
@@ -24,7 +24,9 @@ def __init__(
         self._data_source = data_source
         self._co2_signal_api_token = co2_signal_api_token
 
-    def get_cloud_emissions(self, energy: Energy, cloud: CloudMetadata) -> float:
+    def get_cloud_emissions(
+        self, energy: Energy, cloud: CloudMetadata, geo=None
+    ) -> float:
         """
         Computes emissions for cloud infra
         :param energy: Mean power consumption of the process (kWh)
```
Comment (inimaz) [inline codecarbon/core/emissions.py:32]: add the doc of the param geo here. It is a boolean I guess?

File: docs/_sources/faq.rst.txt [hunk 2]
```diff
@@ -6,11 +6,32 @@ Frequently Asked Questions
...
 	It is hard to quantify the entirety of computing emissions, because there are many factors in play, notably the life-cycle emissions of computing infrastructure. We therefore only focus on the direct emissions produced by running the actual code, but recognize that there is much work to be done to improve this estimation.
 
+* **What are the sources of your energy carbon intensity data?**
+	We use the following sources:
+	
+		For cloud computing :
+
+		- Google publish carbon intensity of electricity for `Google Cloud Plateform <https://cloud.google.com/sustainability/region-carbon?hl=fr>`_.
+
+		- Unfortunately, Amazon has made a habit of keeping information about its carbon footprint out of public view1. Although it released its global carbon footprint, it does not publish datacenter carbon footprints.
```
Comment (inimaz) [inline docs/_sources/faq.rst.txt:16]: typo in `view`

**Review by inimaz — CHANGES_REQUESTED** [review]
Very cool, this will hopefully fix all the intermittent failing tests that we've been having lately in the pipeline. I added 2 small comments if you can check them?

File: docs/_sources/faq.rst.txt (hunk 2 above)
Comment (benoit-cty) [reply docs/_sources/faq.rst.txt]: Thanks, I have fixed it.

File: codecarbon/core/emissions.py (hunk 1 above)
Comment (benoit-cty) [reply codecarbon/core/emissions.py:32]: No, it's a GeoMetadata, I've updated the typing and the docstring.
Thanks for noticing it.

**Review by inimaz — APPROVED** [review]

**Comment by wielrenner** [comment]
"Remove AWS and Azure as they do not provide data."
Any ideas when this can be added back again? Specifically interested in Azure.

**Comment by benoit-cty** [comment]
Hello, if you know a place where we could find the emissions we will be happy to add them as we did not find them.
The original values in CodeCarbon where made by running CodeCarbon on each regions, so it was not representing the real values.

**Pull request merged**